### ISSUE TITLE
Custom popups

### ISSUE THREAD

**Issue by hAMMERlized** [neilimixamo/Home-Assistant-Quick-Look-Mobile#8]
Hi @neilimixamo ,

sorry if i am already annoying you with the requests but as longer i am working with your code the more ideas i got :)

How about some custom popups instead the ugly basic ones from home assistant? Esspecially for lights, cliamte, fan or media player?

Maybe something like or even these, so you don't have to code it yourself:

![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/5aedd985-b3aa-42a8-bd1a-968a0cab5505)

![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/b88872bc-e0ef-4a8f-8e21-86c6d917e718)

![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/9a9c5d44-9e37-415e-b48a-1edb1df6125e)

![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/75a7b354-cd77-4ed0-8568-d315993cc6c0)

Source is the ui-lovelace-minimalist page in the popup section:
https://ui-lovelace-minimalist.github.io/UI/usage/popups/popup_thermostat/

**Comment by neilimixamo** [comment]
I appreciate your suggestions for improvement, and I'm actually delighted to hear them. You're right that the current dialogues for climate and media_player entities may not be visually appealing. However, @matthiasdebaat and @piitaya from NabuCasa have been actively working on redesigning the more-info dialogs since the 2023.3 release, and they have made remarkable progress. It's highly likely that a future Home Assistant update will incorporate this feature. In the meantime, as a temporary solution, we could explore the possibility of using a customized popup in the style of Minimalist UI.

**Comment by hAMMERlized** [comment]
Yea of course i have seen the progress with the basic popups and i appreciate that. But on the other hand, using some custom or even self made popups gives you some kind of independence regarding the popups. 

Anyway, maybe thats just something that concerns me and the rest is just fine with that ;) 

I would implement it myself if i just could but unfortunately thats not my skill. That's why i am glad that i have found your dashboard which makes it massively easier to customize it the way i like. Again, thanks for that!

**Comment by neilimixamo** [comment]
Absolutely, I'd be pleased to help you create your dream dashboard, even if it means crafting some unique views that may not appeal to everyone. I'm currently fine-tuning my 2.0.0 update, but let's definitely revisit your customizations once I'm ready. Feel free to gather all your ideas in the meantime!

**Comment by hAMMERlized** [comment]
Yeah it's all good. The popup function does not hurry as I am not using it that often. 

But tbh a icon / climate counter in the header with the help of a badge or something like that near to the main icon would be really really cool and helpful. Like this it just takes a second to see how man light / climates / windows etc. are on / open / playing and so on.

Can't wait to test the new version. Maybe upload some screens if it's ready and launched 😀 

Thanks for your effort once again!

**Comment by neilimixamo** [comment]
Here's a preview with the dark mode enabled and a more informative header : 

<img src="https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/43101688/7d2cda6b-cf16-43cc-ad85-c94ddefe4b53" width=40% height=40%> <img src="https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/43101688/3de15357-3054-4420-a571-cf4b9d442a33" width=40% height=40%>

**Comment by hAMMERlized** [comment]
Uuhhn, can't wait for it. How about some kind of counting badges in the header for lights / climate / presence / media player etc for quick overview?

Maybe something like that, just "nicer" or "more professional" as I did that only with mushroom cards.

![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/a22c702f-3f1b-4031-a6c1-14a6eb3e15d5)

Regards,
Sebastian

**Comment by neilimixamo** [comment]
Having badges in the Header view could be a great idea. It wouldn't be an issue for me with the Lights view, but I'm unsure about how to proceed with other views like the Air view, where the icon will change based on the active devices. For example, if it's displaying a thermometer icon, the badge could show the number of climates turned on, and if it's a fan icon, it could display the the number of fans turned on. However, the view won't be capable of showing those two badges simultaneously. Would this be ok for you ?

**Comment by hAMMERlized** [comment]
Ah. Yeah true. Forgot the changing icons based on the device turned on.

Hmm how about implementing a variable to turn on / off the changing icons and to chose what icon should be shown or what entitie should be counted? 

Or if that's too complex your solution is also good, or even better. Just thinking loud.
